Extensions/P2P/JsExtension.js

```diff
@@ -0,0 +1,297 @@
+        _('Trigger event _PARAM1_ on client _PARAM0_ (extra data: _PARAM2_)'),
+        _('P2P (experimental)'),
+        'JsPlatform/Extensions/p2picon.svg',
+        'JsPlatform/Extensions/p2picon.svg'
+      )
+      .addParameter('string', _('ID of the other client'), '', false)
+      .addParameter('string', _('Event name'), '', false)
+      .addParameter('scenevar', _('Variable containing the extra data'), '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/P2P/A_peer.js')
+      .addIncludeFile('Extensions/P2P/B_p2ptools.js')
+      .setFunctionName('gdjs.evtTools.p2p.sendVariableTo');
+
+    extension
+      .addAction(
+        'GetEventVariable',
+        _('Get event data (variable)'),
+        _(
+          'Store the data of the specified event in a variable. ' + 
+          'Check in the conditions that the event was received using the "Event received" condition.'
+        ),
+        _('Overwrite _PARAM1_ with variable sent with last trigger of _PARAM0_'),
+        _('P2P (experimental)'),
+        'JsPlatform/Extensions/p2picon.svg',
+        'JsPlatform/Extensions/p2picon.svg'
+      )
+      .addParameter('string', _('Event name'), '', false)
+      .addParameter('scenevar', _('Variable where to store the received data'), '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/P2P/A_peer.js')
+      .addIncludeFile('Extensions/P2P/B_p2ptools.js')
+      .setFunctionName('gdjs.evtTools.p2p.getEventVariable');
+
+    extension
+      .addStrExpression(
+        'GetEventData',
+        _('Get event data'),
+        _('Returns the data received when the specified event was last triggered'),
+        _('P2P (experimental)'),
+        'JsPlatform/Extensions/p2picon.svg'
+      )
+      .addParameter('string', _('Event name'), '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/P2P/A_peer.js')
+      .addIncludeFile('Extensions/P2P/B_p2ptools.js')
+      .setFunctionName('gdjs.evtTools.p2p.getEventData');
+
+    extension
+      .addStrExpression(
+        'GetID',
+        _('Get client ID'),
+        _('Gets the current client ID of the current game instance'),
+        _('P2P (experimental)'),
+        'JsPlatform/Extensions/p2picon.svg'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/P2P/A_peer.js')
+      .addIncludeFile('Extensions/P2P/B_p2ptools.js')
+      .setFunctionName('gdjs.evtTools.p2p.getCurrentId');
+
+    extension
+      .addStrExpression(
+        'GetError',
+        _('Get last error'),
+        _('Gets the description of the last P2P error'),
+        _('P2P (experimental)'),
+        'JsPlatform/Extensions/p2picon.svg'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/P2P/A_peer.js')
+      .addIncludeFile('Extensions/P2P/B_p2ptools.js')
+      .setFunctionName('gdjs.evtTools.p2p.getLastError');
+
+    extension
+      .addStrExpression(
+        'GetDisconnectedPeer',
```

Comment:
`GetLastDisconnectedPeer` maybe? What do you think?

Reply:
True